services/root.js

```diff
@@ -1,7 +1,23 @@
-import { generateApiClient } from "@app/utils/apiUtils";
-const itunesApi = generateApiClient("itunes");
+// import { generateApiClient } from "@app/utils/apiUtils";
+// const itunesApi = generateApiClient("itunes");
 
-export const getRecommendations = async () => {
-  const res = await itunesApi.get(`search?term=`);
-  console.log("res", res);
-};
+// export const getRecommendations = async () => {
+//   const res = await itunesApi.get(`search?term=`);
+//   console.log("res", res);
+//   const getData = response => {
+//     if (!response.ok) {
+//       console.log(res.data);
+//       return [];
+//     }
+
+//     const recomendations = ["arijit", "anuv"];
+//     return recomendations;
+//     // return response.data.filter(({ name }) =>
+//     //   recomendations.includes(name).map(({ id, name }) => ({ id, name }))
+//     // );
+//   };
+
+//   return getData(res);
+// };
+
+/******* work on recommendations later   **********/
```

Comment:
It's good to keep the code that you wanna work later upon commented. But there is good convention.

```js
// @TODO - Work on recommendations later
```

Now for all the `todos` in your application all you do search for `@TODO` and you'll get all the stuff that you need to work upon.

Reply:
Thanks for this information. Will update it accordingly.